integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -1948,6 +1950,88 @@ fn test_reject_block_headers_during_epoch_sync() {
     );
 }
 
+#[test]
```

Comment:
We should not really be using this test framework anymore, see epoch_sync.rs for TestLoop-based tests. TestEnv does not exercise the full code especially when we're dealing with syncs (where the code is very heavily interdependent).

But I see that your test here is very limited in scope and you're asserting the specific error, so let's leave this as is. Other more complex changes you may do would be better tested with TestLoop.